server/http.go

```diff
@@ -419,7 +441,12 @@ func (p *Plugin) getMeetingURL(meetingID int) string {
 }
 
 func (p *Plugin) postConfirm(meetingID int, channelID string, topic string, userID string, creatorName string) *model.Post {
-	meetingURL := p.getMeetingURL(meetingID)
+	creator, _ := p.API.GetUserByUsername(creatorName)
```

Comment:
Why does this error get ignored?

Reply:
If I remember correctly, the creator may be omitted in previous plugin versions. Also, in any case, any error here will default to the base meeting URL, which will help the user. We could log the error happening here, but not 100% sure how useful will be.

Reply:
`postConfirm` is only called when a new meeting gets created. Hence, `meeting_creator_username` should always be set, as the recent meeting was most likely created with the same plugin version. There is a 30 second gap, but I would just ignore that.

Reply:
I will log debug the error, but I keep the handling an "nonexistent user" the same way it is now.